src/java.base/share/native/libzip/zlib/inftrees.h

```diff
@@ -62,7 +62,7 @@ typedef struct {
 /* Maximum size of the dynamic table.  The maximum number of code structures is
    1444, which is the sum of 852 for literal/length codes and 592 for distance
    codes.  These values were found by exhaustive searches using the program
-   examples/enough.c found in the zlib distribtution.  The arguments to that
+   examples/enough.c found in the zlib distribution.  The arguments to that
```

Comment:
This is 3rd party code so should be dropped from the patch as we want as few changes to this code has possible.